update.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
perhaps add -e to this line (or put `set -e` on line 2 or something) so that the script exits if one of its subcommands fails